<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>PDG_LocalisationCreator_GUIClass</class>
  <widget class="QMainWindow" name="PDG_LocalisationCreator_GUIClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>490</width>
        <height>260</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>490</width>
        <height>260</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>490</width>
        <height>260</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>PDG Localisation Tool v2.0</string>
    </property>
    <property name="windowIcon">
      <iconset>
        <normaloff>E:/OneDrive/Desktop/PDGIco.png</normaloff>E:/OneDrive/Desktop/PDGIco.png
      </iconset>
    </property>
    <property name="iconSize">
      <size>
        <width>64</width>
        <height>64</height>
      </size>
    </property>
    <widget class="QWidget" name="centralWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
          <widget class="QFrame" name="folderSelectionBox">
            <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
                <widget class="QLabel" name="outputPathLabel">
                  <property name="text">
                    <string>Output Folder:</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QLineEdit" name="outputPathLineEdit"/>
              </item>
              <item row="0" column="2">
                <widget class="QPushButton" name="outputPathButton">
                  <property name="text">
                    <string>Browse...</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="0">
                <widget class="QLabel" name="vanillaPathLabel">
                  <property name="text">
                    <string>Vanilla Files Folder:</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <widget class="QLineEdit" name="vanillaPathLineEdit"/>
              </item>
              <item row="1" column="2">
                <widget class="QPushButton" name="vanillaPathButton">
                  <property name="text">
                    <string>Browse...</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
        <item>
          <widget class="QFrame" name="actionBox">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
                <widget class="QPushButton" name="unifiedRunButton">
                  <property name="text">
                    <string>ENGAGE</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
        <item>
          <widget class="QGroupBox" name="progressBarGroupBox">
            <property name="title">
              <string>Progress</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_progressBar">
              <property name="bottomMargin">
                <number>0</number>
              </property>
              <item>
                <widget class="QProgressBar" name="progressBar">
                  <property name="value">
                    <number>0</number>
                  </property>
                  <property name="textVisible">
                    <bool>true</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
      </layout>
    </widget>
    <widget class="QStatusBar" name="statusBar"/>
  </widget>
  <resources/>
  <connections/>
</ui>